expression/bench_test.go

```diff
@@ -385,9 +369,9 @@ func genVecExprBenchCase(ctx sessionctx.Context, funcName string, testCase vecEx
 	return expr, input, output
 }
 
-func (s *testEvaluatorSuite) TestVectorizedEvalOneVec(c *C) {
+func (s *testEvaluatorSuite) testVectorizedEvalOneVec(c *C, vecExprCases vecExprBenchCases) {
 	ctx := mock.NewContext()
-	for funcName, testCases := range vecExprBenchCases {
+	for funcName, testCases := range vecExprCases {
```

Comment:
How about supporting specifying test case according to a system environment variable?
For example:
```
>: set vec_test_func_sig=builtinAddDateAndDurationSig
>: go test -v
```
Then only the `builtinAddDateAndDurationSig` will be processed.

Reply:
By running `go test -run=^$ -bench=BenchmarkVectorizedBuiltinMathEvalOneVec`, is it ok after this pr?

Reply:
> By running `go test -run=^$ -bench=BenchmarkVectorizedBuiltinMathEvalOneVec`, is it ok after this pr?

It's ok.